frontend/views/pages/Payments.vue

```diff
@@ -301,6 +314,12 @@ export default ({
         amount: this.withGroupCurrency(amount),
         count: this.paymentsTodo.length
       }
+    },
+    isCloseToDistributionTime () {
+      const dDay = new Date(this.groupSettings.distributionDate)
```

Comment:
`this.groupSettings.distributionDate` is the date for the current period.

The value we want here is for the next period. See how `nextDistribution()` works to get that date.